src/range.rs

```diff
@@ -1,30 +1,222 @@
+//! Version range and requirements data and functions (used for version comparision)
```

Comment:
Let's throw a `.` at the end of these introduction lines in all the modules, structures and methods to be consistent.